gui/wxpython/gui_core/preferences.py

```diff
@@ -1129,7 +1129,7 @@ def _createCmdPage(self, notebook):
         close = wx.CheckBox(
             parent=panel,
             id=wx.ID_ANY,
-            label=_("Close dialog when module is successfully finished"),
+            label=_("Close dialog when command is successfully finished"),
```

Comment:
Should be tool here as well?

Reply:
Probably yes, but I was not sure about formulation. The sentence "tool is successfully finished" seems to me a bit strange. The tool is not a process.
What about changing the sentence to : Close dialog when tool process is completed successfully?

Reply:
How about "Close dialog when completed successfully"?

Reply:
@lindakladivova Please modify your PR accordingly.